Translate/En/21-08.md

```diff
@@ -1,118 +1,121 @@
+As Asuna was being torn by frustration and despair, she noticed a strange phenomenon.
+Oon the ground where dark gravel has been drawn until a while ago, white fog is spreading thinly on top of it. Sliding like ribbons made of thin silk, they under Asuna's as it continues to spread backwards. At the same time, a refreshing sweet scent starts to tickle her nostrils.
 
-_This ... ... The fragrance of the rose ...?_
+_This...The fragrance of the rose...?_
 
-Asuna and Solterina traced the mist ribbon with gaze.
+Asuna and Solterina's gaze traces the ribbon-like mist.
 
-And at the moment when I saw the source, I got a faint sigh from my lips.
+And at the moment when I saw the source, I let out faint sigh from my lips.
 
 "Ah……"
 
 once again.
 
 "Ah"
 
-The source of fog was a young adolescent lying a few tens of meters away.
+The source of the fog was a young teenager lying a few tens of meters away.
 
-To be precise, it is a blue long sword, grasped with his left hand. The blade is broken from the middle, but the mist wraps the whole, it seems as if it emits light faintly.
+To be precise, it is the long blue sword, grasped with his left hand. The blade is broken from the middle, but the mist wraps its whole, it seems as if it emits light faintly.
 
-"Kirito-kun"
+"Kirito-kun."
 
-At the same time Asuna calls the name of the person who loves more than anyone in the shivering voice.
+At the same time Asuna calls the name of the person who she loves more than anyone in a shivering voice.
 
-In the case of
+In the case,
 
-"That “sword of mind” ... ... Kirito’s ..."
+"That “sword of mind” just now, is Kirito’s..."
 
 Impressed, Solterina murmured alike.
 
-The white fog reaches the feet of the Allied Chinese-Korean Army that surrounds the periphery, and it continues to spread still. Although they do not seem to be noticing in battle, they are already being swallowed by pure white ribbons around the knees. At last, it seemed that POH who kept on laughing also noticed something strange.
+The white fog reaches the feet of the Allied Chinese-Korean Army that surrounds the peripherial space, and it still continues to spread. Although they do not seem to notice it because as they are in the midst battle, they are already being swallowed by pure white ribbons around the knees. At last, it seemed that POH who kept on laughing has also noticed something strange.
 
-First of all, I stare at the footsteps and then see Kirito behind like it was played. Do not make a tall person strong for a moment, if you rotate a friend knife kitchen knife on the right and grip it with a rush, it gets to a great deal of trouble.
+First of all, he lowered his gaze towards the footsteps, and then, he saw Kirito behind him. Wasting no time, he threw his "Friendship Cleaver" in his right hand and, but it was easily intercepted. 
```

Comment:
PoH standing in ~5 steps from Kirito, so he raises the "knife", and 
"...should he run and hit, the damage would be extreme..."
There is no hit done yet.